test/bin/scenario.sh

```diff
@@ -14,7 +14,7 @@ SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 # shellcheck source=test/bin/common.sh
 source "${SCRIPTDIR}/common.sh"
 
-DEFAULT_BOOT_BLUEPRINT="rhel-9.2"
+DEFAULT_BOOT_BLUEPRINT="rhel-9.4"
```

Comment:
I reverted it back to `rhel-9.2` and the issue with "forgetting first boot" is gone. There might be something wrong with the ISO or the rhel-9.4's anaconda/kickstart.

I think we should change it back to `rhel-9.2` and investigate in another PR